console/coloring.rst

```diff
@@ -40,13 +40,18 @@ It is possible to define your own styles using the
     use Symfony\Component\Console\Formatter\OutputFormatterStyle;
 
     // ...
-    $outputStyle = new OutputFormatterStyle('red', 'yellow', ['bold', 'blink']);
+    $outputStyle = new OutputFormatterStyle('#F00', 'yellow', ['bold', 'blink']);
```

Comment:
Maybe you should mentioned that this only works for sf5.2+ no?

Reply:
That's why is this pull request made into 5.x branch instead of lower version

Reply:
Yes of course, I meant adding a version added docbloc for this version :)

Reply:
sorry I didn't knew that. I added that `versionadded` and switched red back to previous and changed yellow to hex code. Or do you prefer duplicating that piece of code with note that since 5.2 it can be defined as hexa?